.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish 📦 to PyPi 🚀
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
I just noticed that this project doesn't prefix the tags with `v`. For sake of consistency, i suggest remove the `v`

Reply:
true, will do so